Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -192,6 +192,15 @@ extension IntroViewController: OnboardingCardDelegate {
         }
     }
 
+    // Extra step to make sure pageControl.currentPage is the right index card
+    // because UIPageViewControllerDataSource call fails
```

Comment:
Which call is failing? Nit, but maybe we could specify that?

Reply:
I will address the comment in my other PR because the build is green 🎉